src/mbed_tools/targets/_internal/data/targets_metadata.json

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
Should we remove `targets_metadata.json` in a separate PR?

Reply:
IIRC `mbed-targets` now `.targets` requires it. I'll investigate that and raise another PR if we can remove it.

Reply:
OK

Reply:
I made a sticky note for it